index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,900;1,100;1,300&display=swap"
+      rel="stylesheet"
+    />
+    <title>Gideon's Portfolio</title>
+  </head>
+
+  <body>
+    <div class="template5-main-mobile">
+      <div class="app-bar">
+        <div class="time"><img src="Time.png" alt="time" /></div>
+        <img class="name" src="name.png" alt="name " />
```

Comment:
- [ ] Instead of using an image, here we have to use `a` tag for displaying name as logo.
- **Requirement** The logo (if the design has one) is on the left, and has a link (`a` tag).